packages/vats/src/core/startWalletFactory.js

```diff
@@ -128,6 +128,7 @@ const startGovernedInstance = async (
  */
 export const startWalletFactory = async (
   {
+    vatParameters: { chainStorageEntries = [] },
```

Comment:
Please add `exportStorageSubtrees` to swingset config files such as `decentral-devnet-config.json` to cause this to get populated.

I neglected to do that for PSM too. Would you please cherry-pick this one?

1fd3bc9fce435b3e5f0c09a1de5541778eec49a7